File: Task1/main.cpp
#include "library.h"

int main() {
	initialize_warehouse();
	std::string command_line;
	while (true) {
		std::cout << "> ";
		std::getline(std::cin, command_line);
		if (command_line == "EXIT") {
			break;
		}
		process_command(command_line);
	}
	return 0;
}

File: Task1/Java/Main.java
// Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Library.initializeWarehouse();
        Scanner scanner = new Scanner(System.in);
        String commandLine;

        while (true) {
            System.out.print("> ");
            commandLine = scanner.nextLine();
            if (commandLine.equalsIgnoreCase("EXIT")) {
                break;
            }
            Library.processCommand(commandLine);
        }
        scanner.close();
    }
}
